nessie/externals/s3.py

```diff
@@ -102,7 +134,7 @@ def upload_from_url(url, s3_key):
                 for chunk in response.iter_content(chunk_size=1024):
                     s3_out.write(chunk)
         except (ClientError, ConnectionError, ValueError) as e:
-            app.logger.error(f'Error on S3 upload: source_url={url}, s3_url={s3_url}, error={e}')
+            app.logger.error(f'Error on S3 upload: source_url={url}, bucket={bucket}, key={s3_key}, error={e}')
             return False
-    app.logger.info(f'S3 upload complete: source_url={url}, s3_url={s3_url}')
+    app.logger.info(f'S3 upload complete: source_url={url}, bucket={bucket}, key={s3_key}')
```

Comment:
Logging changed here and above since the logged S3 URL contains full AWS credentials, ulp.